collections/load-testing/index.md

```diff
@@ -1,6 +1,7 @@
 ---
 items:
  - americanexpress/baton
+ - artilleryio/artillery
```

Comment:
Usually we ask people to add new items to the _end_ of an `items:` list (to keep everyone from fighting for the top spots 😅 ). It looks like these are all in alphabetical order though, so I think this is fine 👍🏻